Prace_domowe/Praca_domowa4/Paulina_Jaszczuk/hw4.ipynb

```diff
@@ -0,0 +1,1185 @@
+{
```

Comment:
<p>de facto policzono RMSE a nie MSE (bo squared=False)</p><p> </p><p>dobrze byłoby odnieść RMSE do średniej wartości targetu aby wiedzieć czy to dużo czy nie</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>